web/src/views/settings/ClassificationSettingsView.tsx

```diff
@@ -170,31 +170,22 @@ export default function ClassificationSettingsView({
         );
       })
       .finally(() => {
+        addMessage(
+          "search_settings",
+          `Unsaved Classification settings changes`,
+          undefined,
+          "search_settings",
+        );
         setIsLoading(false);
       });
-  }, [updateConfig, classificationSettings, t]);
+  }, [classificationSettings, t, addMessage, updateConfig]);
 
   const onCancel = useCallback(() => {
     setClassificationSettings(origSearchSettings);
     setChangedValue(false);
     removeMessage("search_settings", "search_settings");
   }, [origSearchSettings, removeMessage]);
 
-  useEffect(() => {
```

Comment:
Not sure if you meant to remove this section, it was working correctly to show a footer message when a user navigated away from the pane without saving first.

Reply:
good catch